packages/compass/src/main/auto-update-manager.ts

```diff
@@ -534,13 +534,9 @@ class CompassAutoUpdateManager {
     compassApp: typeof CompassApplication,
     options: Partial<AutoUpdateManagerOptions> = {}
   ): void {
-    compassApp.addExitHandler(async () => {
+    compassApp.addExitHandler(() => {
       this.stop();
-      try {
-        await this.currentStateTransition;
```

Comment:
Removing this `await` because this seems to break e2e tests for some reason: seems like the way webdriverio quits the app in e2e just makes this hang forever without exiting, I can't reproduce this outside of e2e setup and we don't have logs for main process so it's really hard to debug what it gets stuck on